### PR TITLE
Improvements to the Makefile

### PR DESCRIPTION
**Pull request by AnthonyRobertson17**
ignore the branch name

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+.PHONY: help
+
+help: ## Show all the available make commands
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-10s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+build: ## build the docker image
 	docker build . --tag metric-boy
 
-server:
+server: build ## run the service in the forground
```
Comment (woodgern) [inline Makefile]: It's spelt 'foreground' :P

**Review by woodgern — APPROVED** [review]
The typo fix is only a suggestion, merge at will

File: Makefile (hunk 1 above)
Comment (AnthonyRobertson17) [inline Makefile]:
```suggestion
server: build ## run the service in the foreground
```

**Comment by woodgern** [comment]
Holy shit it happened

**Comment by AnthonyRobertson17** [comment]
Forgot about the PR but then switched to using my personal github for work and saw it again 😂

**Pull request merged**